client/src/pages/wizard/Onboard3.tsx

```diff
@@ -34,18 +35,38 @@ const Onboard3: FC<IProps> = ({ races, setRaces }): ReactElement => {
 
   // All of the race buttons to show in the first row
   const raceButtonsRow1 = [
-    { value: 'American Indian or Alaskan Native', color: '#EF8B8B' },
-    { value: 'Black or African American', color: '#A5C4F2' },
-    { value: 'Middle Eastern or North African', color: '#BAB9E9' },
-    { value: 'Native Hawaiian or Pacific Islander', color: '#A9D3E5' },
+    {
+      display: 'American Indian or Alaskan Native',
```

Comment:
What's the purpose of adding separate display and value fields here? Is it so you don't have to convert the value to uppercase before storing it in the db?

Reply:
Mm kind of. I think it's just better to use the enum itself for the value instead of converting to uppercase